src/style.css

```diff
@@ -0,0 +1,147 @@
+html,
+body,
+div,
+a,
+ul,
+p,
+li,
+header,
+nav {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --mane-text-color: hsl(234, 80%, 12%);
+  --secondary-text-color: hsl(244, 8%, 41%);
+  --button-color: hsl(193, 100%, 43%);
+}
+
+.Card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  height: 408px;
+  background: hsl(0, 0%, 100%);
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  padding: 32px 16px 16px;
+}
+
+.Card-Image {
+  width: 160px;
+  height: 134px;
+  display: block;
+}
+
+.Card-ProductData {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 100%;
+  margin: 40px 0 16px;
```

Comment:
Be consistent with your margins. Add only top or bottom, don't add both.
10 point from [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

Reply:
Done